src/store/flow/block.ts

```diff
@@ -112,18 +112,27 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
     const firstBlock = findBlockOnActiveFlowWith(first.blockId, state as unknown as IContext)
     const secondBlock = findBlockOnActiveFlowWith(second.blockId, state as unknown as IContext)
 
-    const {destinationBlock: firstDesinationBlockId} = findBlockExitWith(first.exitId, firstBlock)
-    const {destinationBlock: secondDesinationBlockId} = findBlockExitWith(second.exitId, secondBlock)
+    const {destinationBlock: firstDestinationBlockId} = findBlockExitWith(first.exitId, firstBlock)
+    const {destinationBlock: secondDestinationBlockId} = findBlockExitWith(second.exitId, secondBlock)
+
+
+
+
+    // todo: this works only when the exit we're targetting is empty
```

Comment:
@safydy-r I will have to fix this in the future; I have it on my todo list. I'm going to leave these messy comments in here for now? 🤷‍♀️

Reply:
it's ok I think